packages/beacon-node/src/util/sszBytes.ts

```diff
@@ -28,29 +28,58 @@ export type AttDataHash = string;
 const ATTESTATION_SLOT_OFFSET = 4;
 const ATTESTATION_BEACON_BLOCK_ROOT_OFFSET = ATTESTATION_SLOT_OFFSET + 8 + 8;
 const ROOT_SIZE = 32;
+const SLOT_SIZE = 8;
+const ATTESTATION_DATA_SIZE = 128;
 
 export function getSlotFromAttestationSerialized(data: Uint8Array): Slot {
+  if (data.length < ATTESTATION_SLOT_OFFSET + SLOT_SIZE) {
+    throw Error(`Invalid attestation data length ${data.length}`);
```

Comment:
Since there is a single failure mode and this is a hot path let's just return null here and document in the JSDocs that null means data too short. Since all of these are SSZ objects we can compute the min size ahead of time and enforce it higher up in the stack, such as in gossip snappy decode step. What do you think? meaning to do both

Reply:
it needs to do after gossip snappy decode step, I think we should do at Lodestar side since we have `sszType.minSize`, `gossipsub.onGossipsubMessage` would be a good place

> return null here and document in the JSDocs that null means data too short.

yeah that's good to do 👍

Reply:
just checked our `DataTransrormSnappy`, it's convenient to check ssz type `minSize/maxSize` there